modules/haloIdSystem.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * This module adds Audigent HaloId to the User ID module
+ * The {@link module:modules/userId} module is required
+ * @module modules/haloIdSystem
+ * @requires module:modules/userId
+ */
+
+import {submodule} from '../src/hook.js';
+import { getStorageManager } from '../src/storageManager.js';
+
+const storage = getStorageManager();
+
+/** @type {Submodule} */
+export const haloIdSubmodule = {
+  /**
+   * Used to link submodule with config
+   * @type {string}
+   */
+  name: 'haloId',
+  /**
+   * Decode the stored id value for passing to bid requests
+   * @function
+   * @param {string} value
+   * @returns {{haloid:string}}
+   */
+  getHaloId(callback) {
+    if (storage.getDataFromLocalStorage('haloId')) {
```

Comment:
what's the difference between these two local storage names, `haloId` and `auHaloId`?